tests/migrations/test_operations.py

```diff
@@ -199,6 +199,42 @@ def test_create_model(self):
         definition = operation.deconstruct()
         self.assertNotIn('managers', definition[2])
 
+    def test_create_model_with_duplicate_field_name(self):
+        with self.assertRaisesMessage(ValueError, "Found duplicate field in CreateModel operation."):
+            migrations.CreateModel(
+                "Pony",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("pink", models.TextField()),
+                    ("pink", models.IntegerField(default=1)),
+                ],
+            )
+
+    def test_create_model_with_duplicate_base(self):
+        with self.assertRaisesMessage(ValueError, "Found duplicate base in CreateModel operation."):
+            migrations.CreateModel(
+                "Pony",
+                fields=[],
+                bases=("test_crmo.Pony", "test_crmo.Pony",),
+            )
+        with self.assertRaisesMessage(ValueError, "Found duplicate base in CreateModel operation."):
+            migrations.CreateModel(
+                "Pony",
+                fields=[],
+                bases=(CustomModelBase, CustomModelBase,),
```

Comment:
The `CustomModelBase` class is not an actual `models.Model` subclass but a `models.base.ModelBase` subclass which is `type(models.Model)` (`models.Model`'s metaclass). We should use an actual `models.Model` subclass instead.